frappe/erpnext
# fix: ignore duplicates explicitly

ref: https://github.com/frappe/frappe/pull/15906 


To test:
1. Check if HSN codes are created on new site
2. Check if Indian company setup goes through. (and has Indian doctypes related to taxation and HR are created )
